I18nConfigInitializer is not exported

**Describe the bug**
I am trying to implement a custom `I18nConfigInitializer` to be able to set a custom dynamic `backend.loadPath` based on the current environment. The problem is, that the `I18nConfigInitializer` is not exported properly.

**Tell us the version of Spartacus**
- Library version: 4.2.0

**To Reproduce**
Steps to reproduce the behavior:
1. Create a custom Config initializer, which extends `I18nConfigInitializer`
2. Use it in the app module providers
3. Notice the console error `Error: Module not found: Error: Can't resolve '@spartacus/core/src/i18n/config/i18n-config-initializer' in '/Users/janwidmer/projects/vzug/workspace-spartacus/src/app'`

**Expected behavior**
`I18nConfigInitializer` is properly exported

**Desktop (please complete the following information):**
 - OS: all OS
 - Browser: all Browsers
 - Version: all Versions

**Smartphone (please complete the following information):**
 - Device: all Devices
 - OS: all OS
 - Browser: all Browser
 - Version: all Versions

--- a/projects/core/src/i18n/index.ts
+++ b/projects/core/src/i18n/index.ts
@@ -1,4 +1,5 @@
 export * from './config/i18n-config';
+export * from './config/i18n-config-initializer';
 export * from './date.pipe';
 export * from './numeric.pipe';
 export { I18nModule } from './i18n.module';

